.github/workflows/tests.yml

```diff
@@ -12,26 +12,78 @@ jobs:
   simple_local:
 
     runs-on: ubuntu-latest
-    timeout-minutes: 5
+    timeout-minutes: 15
     strategy:
       matrix:
         python-version: [3.8, 3.9]
-
+    env:
+      LAMMPSVER: 20201029
+      LAMMPSHASH: da9b6ecb5dc20c7614b9a2ef65903225654badfead9feb8937f1fbc6b4c9838e
     steps:
     - uses: actions/checkout@v2
+      with:
+        submodules: true
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Update container
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-suggests --no-install-recommends \
+          build-essential \
+          libfftw3-dev \
+          libopenmpi-dev \
+          make \
+          ninja-build \
+          pkg-config \
+          software-properties-common
+    - uses: actions/cache@v2
+      id: cache-venv
+      with:
+        path: $HOME/testenv
+        key: ${{ runner.os }}-${{ matrix.python-version }}-${LAMMPSHASH}
+    - name: Set up venv
+      if: steps.cache-venv.outputs.cache-hit != 'true'
       run: |
         python -m venv $HOME/testenv
         . $HOME/testenv/bin/activate
         python -m pip install --upgrade pip setuptools wheel
-        pip uninstall -y radical.pilot radical.saga radical.utils
```

Comment:
Actually, we should restore this line below. The cache is collected at the end of the job, so (once the venv cache bundle exists) the restored venv will have arbitrary versions of RCT installed at a later point in a previous run. We should be cleaner...